.github/workflows/prod-docker.yml

```diff
@@ -0,0 +1,112 @@
+name: Production Superhero Backend Pipeline
+
+on:
+  push:
+    branches:
+      - main
+  release:
+    types: [created]
+
+env:
+  ENV: "prd"
+  APP: "superhero-backend"
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    name: Production Pipeline
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 100
+
+      - uses: GoogleCloudPlatform/release-please-action@v3
+        id: release
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          release-type: node
+          package-name: ""
+          changelog-types: '[{"type":"feat","section":"Features","hidden":false},{"type":"fix","section":"Bug Fixes","hidden":false},{"type":"ci","section":"CI / CD","hidden":false},{"type":"test","section":"Testing","hidden":false},{"type":"refactor","section":"Refactorings","hidden":false},{"type":"chore","section":"Miscellaneous","hidden":false}]'
+
+      - name: Set up Docker Buildx
+        id: buildx
+        # Use the action from the master, as we've seen some inconsistencies with @v1
+        # Issue: https://github.com/docker/build-push-action/issues/286
+        uses: docker/setup-buildx-action@master
+        # Only worked for us with this option on
+        with:
+          install: true
+
+      - name: Cache Docker layers
+        uses: actions/cache@v2
+        with:
+          path: /tmp/.buildx-cache
+          # Key is named differently to avoid collision
+          key: ${{ runner.os }}-${{ env.ENV }}-buildx-${{ github.sha }}
+          restore-keys: |
+            ${{ runner.os }}-${{ env.ENV }}-buildx
+
+      - name: Log in to dockerhub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USER }}
+          password: ${{ secrets.DOCKERHUB_PASS }}
+
+      - name: Extract metadata for docker
+        if: |
+          github.event_name == 'push' ||
+          startsWith(github.ref, 'refs/tags/v')
```

Comment:
why not `if: ${{ steps.release.outputs.release_created }}` here?

Reply:
ye, this can be changed , as we are going to run only on release created.

Reply:
i think we can run it on tags